src/WheelsAndGoods.Application/Mapping/ApplicationMapper.cs

```diff
@@ -84,6 +84,43 @@ public UserCreatedResponse ToUserCreatedResponse(AuthResponse authResponse, User
         };
     }
 
+	public Order ToUpdatedOrder(UpdateOrderRequest updateOrderRequest, Guid orderId, User customer)
+	{
+		return new Order(orderId)
+		{
+			Title = updateOrderRequest.Title,
+			Cargo = updateOrderRequest.Cargo,
+			DeliveryDeadlineAtUtc = updateOrderRequest.DeliveryDeadlineAtUtc,
+			Description = updateOrderRequest.Description,
+			From = updateOrderRequest.From,
+			To = updateOrderRequest.To,
+			Price = updateOrderRequest.Price,
+			Customer = customer
+		};
+	}
+
+	public OrderResponse ToOrderResponse(Order order, User user)
```

Comment:
Looks like we already have a method like this but without user. Please have a look if it can be used instead of this added.